addons/bus/models/bus.py

```diff
@@ -193,6 +193,12 @@ def start(self):
         self.started = True
         return self
 
+    def im_status(self, dbname, partner_ids):
```

Comment:
Why is this method needed, why is it public, and why does it run in sudo mode all the time?